File: sample-app/src/main/java/io/github/lazymockbean/sampleapp/service/ExampleServiceA.java
package io.github.lazymockbean.sampleapp.service;

import org.springframework.stereotype.Service;

@Service
public class ExampleServiceA implements ExampleService {
    @Override
    public String getExample1() {
        return "example-A-1";
    }

    @Override
    public String getExample2() {
        return "example-A-2";
    }
}

File: sample-app/src/main/kotlin/io/github/taesk/lazymockbean/sampleapp/service/ExampleServiceA.kt
package io.github.taesk.lazymockbean.sampleapp.service

import org.springframework.stereotype.Service

@Service
class ExampleServiceA : ExampleService {
    override fun getExample1(): String {
        return "example-A-1"
    }

    override fun getExample2(): String {
        return "example-A-2"
    }
}
